client/my-sites/plan-features-2023-grid/components/billing-timeframe.tsx

```diff
@@ -71,16 +72,27 @@ function usePerMonthDescription( {
 				: null;
 
 		if ( fullTermPriceText ) {
+			const isEnglishLocale = currentLocale === 'en' || currentLocale?.startsWith( 'en-' );
+
 			if ( PLAN_ANNUAL_PERIOD === billingPeriod ) {
-				return translate( 'per month, %(fullTermPriceText)s billed annually', {
+				// TODO: Remove check once text is translated
+				return translate( 'per month, %(fullTermPriceText)s billed annually, Excl. Taxes', {
```

Comment:
Can we add a translation comment here to explain that `Excl. Taxes` means "Excluding taxes"? It may not be clear to everyone.

Reply:
Added comments in [ac55cb9](https://github.com/Automattic/wp-calypso/pull/76738/commits/ac55cb9b64286326a9df2a2f0acbc74d32f72412)